src/main/java/Duke.java

```diff
@@ -1,10 +1,318 @@
+    }
+
+    private static void loadFileContents(File F, ArrayList<Task>tasks, ArrayList<String>commandInput,
+                                         boolean canPrintFeedback) throws IOException {
+
+        Scanner s = new Scanner(F);
+        while(s.hasNext()){
+            processUserInput(tasks,commandInput,s.nextLine(),F,canPrintFeedback);
+        }
+    }
+    private static String[] splitCommandAndDescription(String userInput) {
+        String[] commandAndArgs = userInput.split(" ", 2);
+        return commandAndArgs.length == 2 ? commandAndArgs : new String[]{commandAndArgs[0], ""};
+    }
+
+    private static void processUserInput(ArrayList<Task> tasks, ArrayList<String> inputCommands,String userInput,File F,
+                                         boolean canPrintFeedback) {
+        inputCommands.add(userInput);
+        String[] commandAndDescription = splitCommandAndDescription(userInput);
+        String command = commandAndDescription[0];
+        String description = commandAndDescription[1];
+        try {
+            switch (command) {
+            case COMMAND_BYE:
+                printByeMessage();
+                break;
+            case COMMAND_EVENT: //append
+                executeEventCommand(tasks, description,canPrintFeedback);
+                if(canPrintFeedback){
+                    appendToFile(F.getPath(),userInput);
+                }
+                break;
+            case COMMAND_DEADLINE: //append
+                executeDeadlineCommand(tasks, description,canPrintFeedback);
+                if(canPrintFeedback){
+                    appendToFile(F.getPath(),userInput);
+                }
+                break;
+            case COMMAND_LIST:
+                executeListCommand(tasks,canPrintFeedback);
+                break;
+            case COMMAND_TODO: //append
+                executeTodoCommand(tasks, description,canPrintFeedback);
+                if(canPrintFeedback){
+                    appendToFile(F.getPath(),userInput);
+                }
+                break;
+            case COMMAND_MARK: //write
+                executeMarkCommand(tasks, description,canPrintFeedback);
+                if(canPrintFeedback){
+                    writeToFile(F.getPath(), inputCommands);
+                }
+                break;
+            case COMMAND_UNMARK://write
+                executeUnmarkCommand(tasks, description,canPrintFeedback);
+                if(canPrintFeedback){
+                    writeToFile(F.getPath(), inputCommands);
+                }
+                break;
+            case COMMAND_DELETE:
+                executeDeleteCommand(tasks,description);
+                break;
+            default:
+                System.out.println("No such commands found! Please try again!");
+                System.out.println(LINE);
+            }
+        }catch(IOException e){
+            System.out.println("Unable to write/append to file.");
+        }
+
+    }
```

Comment:
Try to have less than 30 LOC for any method. You can refactor the code in this method.